tests/dummy/app/templates/overview.hbs

```diff
@@ -35,7 +37,7 @@
           <small>(only core, widget, mouse, resizable, and sortable modules required)</small>
         </li>
         <li><a target="_BLANK" href="https://github.com/brandonaaron/jquery-mousewheel">jquery.mousewheel</a></li>
-        <li><a target="_BLANK" href="https://github.com/LearnBoost/antiscroll">antiscroll</a></li>
+        <li><a target="_BLANK" href="https://github.com/azirbel/antiscroll">antiscroll</a></li>
```

Comment:
Also it seems like the ownership of the project has been transferred https://github.com/Automattic/antiscroll

Unless we have custom modifications that are not accepted through a PR, we probably want to rely on the "official" one.